kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/parquet/ParquetColumnReaderSuite.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.internal.parquet
+
+import java.util.TimeZone
+
+import io.delta.golden.GoldenTableUtils.goldenTablePath
+import io.delta.kernel.defaults.utils.TestRow
+import io.delta.kernel.types._
+import org.scalatest.funsuite.AnyFunSuite
+
+/**
+ * Suite covering reading Parquet columns with different types.
+ */
+class ParquetColumnReaderSuite  extends AnyFunSuite with ParquetSuiteBase {
```

Comment:
rename to `ParquetTypeWideningSuite` or merge these tests into `ParquetFileReaderSuite.scala`?

Reply:
I moved the tests to `ParquetFileReaderSuite`